nixos/modules/system/boot/kernel.nix

```diff
@@ -108,7 +108,7 @@ in
     boot.extraModulePackages = mkOption {
       type = types.listOf types.package;
       default = [];
-      example = literalExample "[ pkgs.linuxPackages.nvidia_x11 ]";
+      example = literalExample "[ config.boot.kernelPackages.nvidia_x11 ]";
```

Comment:
Can you put something similar to what's in the wiki as the description?
Just a note that you should always use `config.boot.kernelPackages` to ensure it's the correct module set.

Reply:
I'm sorry, but I don't have the time right now to pay attention to this problem. I may revisit this topic a bit later.